### DISCUSSION TITLE
UCTronics not working

### DISCUSSION THREAD

**Discussion by mike2246** [gtxaspec/wz_mini_hacks#553] [Q&A]
How do I edit the config for the following POE Adapter
UCTRONICS for Raspberry Pi Zero Ethernet and Power, Micro USB Ethernet/PoE Adapter for Fire TV Stick, Chromecast, Google Mini, and More, IEEE 802.3af Compliant

I have the following set:
ENABLE_USB_ETH="true"
ENABLE_USB_ETH_MODULE_AUTODETECT="true"
ENABLE_USB_ETH_MODULE_MANUAL=""

**Comment by bjs-pdx** [comment]
A little more information...what camera?  Which firmware version?  What results do you get from the camera?  Does SSH work?  Web interface?

How long have you had the adapter?  Is it one of the older ones missing the MAC?  Have you tested it on another device (Pi, laptop, etc.?)
